package.json

```diff
@@ -81,7 +81,7 @@
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.0.5",
     "husky": "^7.0.2",
-    "jest": "^27.0.6",
+    "jest": "^26.1.0",
```

Comment:
What about Jest was breaking? Isn't it worth trying to address now because we'll have to update it eventually anyways?

Reply:
It was an issue with `react-test-renderer` .. it got a bit messy with all tests and snaps updates I had to do and still couldn't get all errors sorted. I'd prefer to tackle it in a separated PR and leave this just with the Dependabot updates.

Reply:
Understood!